src/api/__tests__/Feed-test.js

```diff
@@ -202,24 +238,14 @@ jest.mock('../Comments', () =>
     })),
 );
 
-jest.mock('../Versions', () =>
-    jest.fn().mockImplementation(() => ({
-        getVersions: jest.fn().mockReturnValue({
-            total_count: 1,
-            entries: [
-                {
-                    action: 'upload',
-                    type: 'file_version',
-                    id: 123,
-                    created_at: 'Thu Sep 20 33658 19:45:39 GMT-0600 (CST)',
-                    trashed_at: 1234567891,
-                    modified_at: 1234567891,
-                    modified_by: { name: 'Akon', id: 11 },
-                },
-            ],
-        }),
-    })),
-);
+jest.mock('../Versions', () => {
+    const version = mockFirstVersion;
+    const currentVersion = mockCurrentVersion;
+    return jest.fn().mockImplementation(() => ({
+        getVersions: jest.fn().mockReturnValue(version),
+        getCurrentVersion: jest.fn().mockReturnValue(currentVersion),
```

Comment:
The shortcut for this should be `jest.fn(() => mockCurrentVersion)`, likewise for the above;